code/compute_fc/compute_fc.py

```diff
@@ -0,0 +1,883 @@
+        "--atlas-dimension",
+        default=64,
+        type=int,
+        help="dimension of the atlas (usually 64, 128 or 512)",
+    )
+    parser.add_argument(
+        "--low-pass",
+        default=0.15,
+        action="store",
+        type=float,
+        help="cutoff frequency of low pass filtering",
+    )
+    parser.add_argument(
+        "--FD-thresh",
+        default=0.4,
+        action="store",
+        type=float,
+        help="framewise displacement threshold (in mm)",
+    )
+    parser.add_argument(
+        "--SDVARS-thresh",
+        default=3,
+        action="store",
+        type=float,
+        help="standardised DVAR threshold",
+    )
+    parser.add_argument(
+        "--n-scrub-frames",
+        default=5,
+        action="store",
+        type=int,
+        help="minimum segment length after volume censoring",
+    )
+    parser.add_argument(
+        "--fc-estimator",
+        default="sparse inverse covariance",
+        action="store",
+        type=str,
+        help="""type of connectivity to compute (can be 'correlation', 'covariance' or
+        'sparse')""",
+    )
+    parser.add_argument(
+        "--no-censor",
+        default=False,
+        action="store_true",
+        help="interpolate volumes with high motion without censoring",
+    )
+
+    parser.add_argument(
+        "-v",
+        "--verbosity",
+        action="count",
+        default=1,
+        help="""increase output verbosity (-v: standard logging infos; -vv: logging
+        infos and NiLearn verbose; -vvv: debug)""",
+    )
+
+    args = parser.parse_args()
+
+    return args
+
+
+def get_func_filenames_bids(paths_to_func_dir, task_filter=[]):
+    logging.debug("Using BIDS to find functional files...")
+
+    layout = BIDSLayout(
+        paths_to_func_dir,
+        validate=False,
+    )
+
+    all_derivatives = layout.get(
+        scope="all",
+        return_type="file",
+        extension="nii.gz",
+        session=[15],
+        suffix="bold",
+        task=task_filter,
+    )
+
+    t_rs = []
+    for file in all_derivatives:
+        t_rs.append(layout.get_metadata(file)["RepetitionTime"])
+
+    unique_tr_s = set(t_rs)
+
+    if len(unique_tr_s) > 1:
+        logging.warning(
+            "Multiple TR values found, temporal filtering may not" " work as intended !"
+        )
+
+    return all_derivatives, list(unique_tr_s)[0]
+
+
+def get_bids_savename(filename, patterns=FC_PATTERN, **kwargs):
+    entity = parse_file_entities(filename)
+
+    for key, value in kwargs.items():
+        entity[key] = value
+
+    bids_savename = build_path(entity, patterns)
+    logging.debug(f"BIDS filename is :\n\t{build_path(entity, patterns)}")
+
+    return bids_savename
+
+
+def get_atlas_data(atlas_name="DiFuMo", **kwargs):
+    logging.info("Fetching the DiFuMo atlas ...")
+
+    if kwargs["dimension"] not in [64, 128, 512]:
+        logging.warning("Dimension for DiFuMo atlas is different from 64, 128 or 512 !")
```

Comment:
Can you add one line in the warning explaining what is the 
```suggestion
        logging.warning("Dimension for DiFuMo atlas is different from 64, 128 or 512 ! Are you certain you want to deviate from those optimized modes? ")
```